packages/odyssey-react-mui/src/Form.tsx

```diff
@@ -95,16 +101,25 @@ const Form = ({
   encodingType,
   formActions,
   id: idOverride,
+  isFullWidth,
   method,
   name,
   noValidate = false,
+  onSubmit,
   target,
   testId,
   title,
   translate,
 }: FormProps) => {
   const id = useUniqueId(idOverride);
 
+  const onFormSubmit: FormEventHandler = (event) => {
+    const formElement = event.target as HTMLFormElement;
+    const formData = new FormData(formElement);
+
+    onSubmit?.(event, formData);
+  };
+
```

Comment:
Not everyone needs this. Most form libraries do it for you. We just need to pass the `onSubmit` straight through.

---

We talked about this in our call, so it should be good now with the changes you made.